README.md

```diff
@@ -122,6 +123,54 @@ From the application directory run:
 bin/console fom:user:resetroot
 ```
 
+## Docker
+
+Run this command to start a local mapbender instance in a docker container using an internal sqlite database.
+
+```bash
+docker run mapbender/mapbender
+```
+
+### external database 
+
+To run a Mapbender docker container configured to use an external database we will use the MAPBENDER_DATABASE_URL environment variable.
+
+1.  Create the docker network
+    `docker network create -d bridge mapbender`
+
+2.  Start the database
+    `docker run --name db --network mapbender -p 55432:5432 -e POSTGRES_USER=pguser -e POSTGRES_PASSWORD=pgpass -e POSTGRES_DB=postgres postgis/postgis:14-3.4`
+
+3.  Start mapbender
+    `docker run --name mapbender --network mapbender -p 80:8080 -e MAPBENDER_DATABASE_URL="pgsql://pguser:pgpass@db:5432/postgres" mapbender/mapbender`
+
+4.  Create Mapbender database tables
+    `docker exec mapbender php application/bin/console doctrine:schema:update --force`
```

Comment:
isn't it 

`bin/console doctrine:schema:create`

https://docs.mapbender.org/develop/en/installation/installation_ubuntu.html

Beim Update einer bestehenden Mapbender db muss nach den Anweisungen unter https://github.com/mapbender/mapbender/blob/develop/UPGRADING.md#upgrade-database erfolgen. 

Vielleicht ein Hinweis darauf?

Reply:
I am actually unsure about this one.
From a glance at the implementation the "create" should rather be an option of the "update" command since it does almost the same.

Reply:
ok. so we ask @Phocacius

Reply:
update will always do the job, whether there is an existing database or not. Create will only work on an empty database. So `update` is a good choice IMO. The only thing it won't do is create the database itself if it doesn't exist yet, but that makes sense to do manually anyway (for access right reasons, otherwise the mapbender user would need to be a superuser on the database)